rll/collectd.sh

```diff
@@ -36,7 +36,7 @@ function run_check_write_needed() {
 # $1: the file path to get the checksum from
 #
 function run_checksum() {
-  sha256sum $1 | cut -d ' ' -f 1
+  sudo sha256sum $1 | cut -d ' ' -f 1
```

Comment:
Since we are adding sudo here to make sure we can access the file passed in, we should also add sudo in `run_check_write_needed` above in checking if the file exists:
```
sudo [ ! -f $1 ] || [[ `run_checksum $2` != `run_checksum $1` ]]
```